### ISSUE TITLE
Using Data binding on viewModels

### ISSUE THREAD

**Issue by ahmetcj4** [n26/N26AndroidSamples#6]
Thank you so much for this sample of Reactive clean mvvm architecture. It is amazing but would be better if binding is done with android databinding, in my opinion. Is there any issue for not using it?

**Comment by luciapayo** [comment]
I agree data binding fits with this pattern very well but is not in my roadmap to implement it soon. In any case, feel free to fork it and try it out! I'll be happy to see how it looks :)
